### ISSUE TITLE
Randomly generated graves return after restart

### ISSUE THREAD

**Issue by MoonCutter2B** [FireBall1725/Graves#65]
Close to my base I found a number of graves that I looted.
I got the loot and the grace was gone, but there is still something invisible/inpassble/un-hittable in the place of the grave. 
And after relogging the grave is back.
It doesn't happen every time. 
It might have been a problem with forge etc. since most of them (all?) are close to my spawn point/base.
It might be a problem you already solved ;)
I don't have any loggs (yet).
I think I can use MCedit or world edit to remove the block for real.

**Comment by FusionLord** [comment]
Logs, Version, Minecraft Version?

**Comment by MoonCutter2B** [comment]
oh, many days ago now 
well it is/was MC 1.10.2
It was a new world where the mod was configured to not spawn ghosts (I died to many time so it wasn't funny any more). 
I don't think this happened at all in my previous 1.10.2 world.
I think I checked the logs right after I opened this issue but I couldn't see anything that was related to the problem.
Since then I have removed the mod (and all invisible graves are gone). And now I have added the mod again but I don't think I'v generated any new chunks so I don't know if the problem is still there or not.
I guess this doesn't help much but I will keep an eye out to see what happens.
